Anlab.Mvc/Views/Lab/Finalize.cshtml

```diff
@@ -60,6 +60,25 @@
             $('#adjustmentAmount').val(adjustAmt);
             $('#adjustAmt').html(adjustAmt);
         });
+
+        $('#UploadFile').on('change', function () {
+
+            var fullFileName = $('#UploadFile').val().replace(/^.*[\\\/]/, '').toUpperCase();
+            var fileName = fullFileName.replace(/\.[^/.]+$/, "");
+            var extension = fullFileName.split('.').pop();
+            var element = $("#UploadFile")[0];
```

Comment:
@jSylvestre can you not get filename from the element?  As in, instead of doing the multiple jquery inspections and just using the element you already got here